### PR TITLE
:arrow_up: Update on README.md

### PR DESCRIPTION
**Pull request by GuilhermeBraz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,21 +29,28 @@ $ git clone https://github.com/AcompanhaLegis/docs.git
 $ cd docs
 </code>
 
-#### 3. Build the docker image from Dockerfile
+#### 3. You must activate the venv bin, it will be activated using
 
 <code>
-$ sudo docker build -t me-adota-docs .
+$ virtualenv venv
+```
```
Comment (nova-script) [inline README.md]: Okay, there's a bug here. I think there's no need of lines 36 and 37.

File: README.md [hunk 2]
```diff
@@ -29,21 +29,28 @@ $ git clone https://github.com/AcompanhaLegis/docs.git
...
+```
```
Comment (nova-script) [inline README.md]: Okay, there's a bug here. I think there's no need of lines 36 and 37.

**Review by nova-script — CHANGES_REQUESTED** [review]
Please, read the commentaries.

**Comment by nova-script** [comment]
Also, please: check the requirements section of the README. There is no need of Docker. Please, remove Dockerfile too.

**Comment by Douglas-Castro** [comment]
Please, add the license link to README.md, @GuilhermeBraz.

**Review by douglasffcastro — CHANGES_REQUESTED** [review]
Please, put the commands in step 3 on different lines.

**Review by douglasffcastro — CHANGES_REQUESTED** [review]
Please, add the license link to README.md.

File: README.md [hunk 3]
```diff
@@ -29,21 +27,26 @@ $ git clone https://github.com/AcompanhaLegis/docs.git
 $ cd docs
 </code>
 
-#### 3. Build the docker image from Dockerfile
+#### 3. You must activate the venv bin, it will be activated using
 
 <code>
-$ sudo docker build -t me-adota-docs .
+$ virtualenv venv
```
Comment (nova-script) [inline README.md]: The two lines are being displayed at one. This should not happen.

**Review by nova-script — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GuilhermeBraz) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (GuilhermeBraz) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (GuilhermeBraz) [reply README.md]: fixed

**Review by nova-script — APPROVED** [review]

**Review by douglasffcastro — APPROVED** [review]

**Pull request merged**